### PR TITLE
Development

### PR DESCRIPTION
**Pull request by cdsandoval**

### CODE REVIEW

File: src/components/DiscussionForm.js [hunk 1]
```diff
@@ -0,0 +1,161 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+
+function DiscussionForm({ setModalIsOpen, setDiscussions, discussions }) {
+  function JSONify(elements) {
+    var obj = {};
+    for (var i = 0; i < elements.length; ++i) {
```
Comment (byteconde) [inline src/components/DiscussionForm.js]: Was it possible to use map?

Comment (valevassallo) [reply src/components/DiscussionForm.js]: Perhaps it was, could be another way to put this

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,50 @@
...
+    <main>
+      <React.Suspense fallback={<Loading />}>
+        <div>
+          <Router>
+            <Header default />
+          </Router>
+        </div>
+        <Router>
+          <LoginForm path="/" onUser={handleUser} />
+          <Main
```
Comment (diegotc86) [inline src/index.js]: Hi!, where do you validate if an user is already logged in?

File: src/index.js [hunk 3]
```diff
@@ -1,12 +1,50 @@
...
+            path="/discussion"
+            discussions={discussions}
+            setDiscussions={setDiscussions}
+          />
+          <Discussion
+            path="/:title"
```
Comment (diegotc86) [inline src/index.js]: What do you think it will happens if a create a discussion with title "discussion"? :D

**Pull request merged**